src/test/debuginfo/function-prologue-stepping-no-stack-check.rs

```diff
@@ -37,100 +37,79 @@
 // gdb-command:run
 
 // IMMEDIATE ARGS
-// gdb-command:print a
-// gdb-check:$1 = 1
-// gdb-command:print b
-// gdb-check:$2 = true
-// gdb-command:print c
-// gdb-check:$3 = 2.5
+// gdb-command:info args
```

Comment:
That's certainly a problem. I don't think I can get this test to work without either this workaround or a generalized solution for #32949.
I'm not entirely sure the test makes any sense anymore, btw, a the whole `morestack` stuff is gone.

I'd rather have a test that only makes sure the backtrace contains the argument names and their values, and it wouldn't need the myriad of cases found here.